vegbilder-client/src/constants/defaultParamters.ts

```diff
@@ -2,6 +2,7 @@ export const DEFAULT_COORDINATES = { lat: 65, lng: 15 };
 export const DEFAULT_ZOOM = 4;
 export const DEFAULT_VIEW = 'map';
 export const DEFAULT_IMAGE_ID = '';
+export const DEFAULT_ROAD_CATEGORIES: string[] = ["E", "R", "F"]
```

Comment:
Eg forstår kvifor du har gjort dette (for å lett kunne skru av og på kva som skal visast i tabellen, sikkert), men det som er litt uheldig slik koden står no er at det ikkje er nokon samanheng mellom det som står her, og typen `IStatisticsRow`. Dermed går det an å f.eks. legge til "K" her, men ikkje i `IStatisticsRow`, og det vil ikkje gi forventa oppførsel. Derfor foreslår eg å droppe denne.

Dersom ein vil begynne å leike seg med avansert typescript, så går det an å heller modellere dette gjennom typesystemet. Ein kunne f.eks. ha hatt ein type

```typescript
type SynligVegkategori = "E" | "R" | "F"
```

Også kunne ein samtidig hatt ein typescript record, som er ein dictionary-type. F.eks. kunne det ha sett slik ut:

```typescript
// Keys er av typen SynligVegkategori, og verdiene er number
type KolonneVerdier = Record<SynligVekategori, number>
```

Men eg forventar **ikkje** at du skal gjere dette no.

Reply:
Det så veldig interessant ut, så det må jeg sjekke ut!

Reply:
Merk at eg ikkje har tenkt 100% gjennom forslaget over, så det er ikkje sikkert at det fører nokon veg, men på generelt grunnlag kan ein gjere veldig mykje med typesystemet til typescript, og det er å føretrekke framfor strings i eit array fordi typesystemet då kan gi garantiar for korleis dataene ser ut, eller feil dersom ein prøvar å bruke det på feil måte.